libraries/app/database_api.cpp

```diff
@@ -115,6 +115,7 @@ class database_api_impl : public std::enable_shared_from_this<database_api_impl>
       market_ticker                      get_ticker( const string& base, const string& quote, bool skip_order_book = false )const;
       market_volume                      get_24_volume( const string& base, const string& quote )const;
       order_book                         get_order_book( const string& base, const string& quote, unsigned limit = 50 )const;
+      vector<market_volume>              get_top_markets(uint limit)const;
```

Comment:
Code style: add spaces.

By the way, I don't like `uint`, perhaps change to `uint32_t`?